main.py

```diff
@@ -37,9 +39,23 @@ def main():
             print("Cleaning up namespace %s" % namespace.metadata.name)
             # real cleanup will be enable later once it's tested
             v1api.delete_namespace(namespace.metadata.name, client.V1DeleteOptions())
+            cleaned += 1
 
     print("Finish clean up script")
-
+    return cleaned
 
 if __name__ == '__main__':
-    main()
+    start = time.time()
+    eventdict = { 'name' : 'msftkube', 'time' : start, 'eventtype' = 'json'}
```

Comment:
Maybe use a name that's a bit less implementation specific

Reply:
whooops. aeva.cleaner was actually my plan this is a copy and paste bug.